content/source/docs/enterprise/before-installing/reference-architecture/vmware.md

```diff
@@ -192,11 +192,12 @@ Should the *TFE-main* server fail, it can
 be recovered, or traffic can be routed to the *TFE-standby* server to
 resume service when the failure is limited to the Terraform Enterprise server layer. See the Disaster Recovery section.
 
-#### Single ESXi Server Failure
+#### Single ESXi Host Failure
 
-In the event of a single ESXi server failure, ESXi will vMotion the Terraform Enterprise virtual
-machine to a functioning ESXi host. This typically does not result in any
-visible outage to the end-user if VM Fault Tolerance has been configured. As with all applications, you may see a small interruption while the VM boots on the new host if Fault Tolerance is not configured.
+In the event of a single ESXi host failure, vSphere HA will restart the Terraform Enterprise virtual
+machine to a functioning ESXi host in the cluster. This restart can take up to 30 seconds for the failed virtual machine to come back online on a healthy host within the cluster.
+
+If VMware vSphere Fault Tolerance (FT) has been configured for the TFE server, the failover does not result in any visiable outage to the end user.
```

Comment:
Yes this would be based on the general recommendation that the customer use vSphere HA at the cluster (This is a VMware base recommendation for any compute cluster) and automated restarts would take place if a host fails. 

We mentioned FT in the original doc (I have rarely seen people use it when I worked at VMware) so I kept it as a 0 downtime type option but this would really be a corner case.

Reply:
OK, that answered my concerns.